packages/reactive-element/src/decorators/query.ts

```diff
@@ -98,15 +98,14 @@ export function query(selector: string, cache?: boolean): QueryDecorator {
             })();
       return desc(protoOrTarget, nameOrContext, {
         get(this: ReactiveElement): V {
-          if (cache) {
-            let result: V = get!.call(this);
-            if (result === undefined) {
-              result = doQuery(this);
+          let result: V = get!.call(this);
```

Comment:
I've removed the `if (cache)` statement as it should always be true at this point. Possibly a copy+paste mistake?

Reply:
good catch!